server/aurora_mcp/resources.py

```
@@ -0,0 +1,135 @@
+logger = logging.getLogger(__name__)
+
+ApiCall = Callable[..., Awaitable[Dict[str, Any]]]
+
+
+async def _do_catalog_connectors(api_call: ApiCall) -> Dict[str, Any]:
+    try:
+        return truncate_payload(
+            await api_call("GET", "/api/connectors/status"),
+            tool_name="catalog/connectors",
+        )
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)